duckduckgo/iOS
# Fix for bookmarks favicon caching

Task/Issue URL: https://app.asana.com/0/414709148257752/1202223699061973/f
CC: @miasma13 

**Description**: When visiting sites ensure that only favicons for bookmarked sites are saved to the bookmarks cache



**Steps to test this PR**:
1. Visit a website that is not bookmarked.
2. Confirm that it's favicon is saved only to the .tabs cache and not to the .bookmarks cache
3. Import bookmarks from HTML file containing some sites that you have not already bookmarked
4. The imported bookmarks will have a default placeholder favicon5. 
5. Tap one of the imported bookmarks to visit that site6. 
6. Return to the bookmarks screen and confirm this bookmark now has a favicon (the same site as visible in tabs view)




---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
